tamr_toolbox/filesystem/cloud.py

```diff
@@ -0,0 +1,141 @@
+from google.cloud.storage import Client as GcsClient
```

Comment:
I don't recall why, but this doesn't work for this module.  You'll need to `from google.cloud import storage` and then refer to `storage.Client`.

Reply:
This is the pattern in the documentation for that package, so it seems reasonable to follow it: https://pypi.org/project/google-cloud-storage/